cinema/urls.py

```diff
@@ -6,7 +6,7 @@
     ActorViewSet,
     CinemaHallViewSet,
     MovieViewSet,
-    MovieSessionViewSet,
+    MovieSessionViewSet, OrderSessionViewSet,
```

Comment:
The `OrderSessionViewSet` is registered in the router. Ensure that this is the correct name for the viewset, as it might be intended to be `OrderViewSet` based on typical naming conventions. Verify the viewset name in the views file to ensure consistency.